packages/shared/src/utils/contract/contract.ts

```diff
@@ -154,3 +154,9 @@ export const getContractUpdateConfig = ({ contract_update, limit_order }: TGetCo
 export const shouldShowExpiration = (symbol: string) => /^cry/.test(symbol);
 
 export const shouldShowCancellation = (symbol: string) => !/^(cry|CRASH|BOOM|stpRNG|WLD|JD)/.test(symbol);
+
+export const getSubType = (type: string) => {
```

Comment:
Type is equal to `TURBOSLONG` or `TURBOSSHORT`, function will return 'Short' or 'Long'